### ISSUE TITLE
Make line mode arrow keys configurable

### ISSUE THREAD

**Issue by excetara2** [acejump/AceJump#234]
Press the AceJump shortcut, followed by → to target the last, ← to target the first, or ↑, to target the first non-whitespace characters of every line in the editor.

Can this be changed to use <ctrl+f>, <ctrl+b>, and <ctrl+p>??

Just from using emacs I am really used to these bindings so would be appreciated to know how to do this.

**Comment by breandan** [comment]
Currently, due to the way we are handling arrow keys this is not possible. But it would be a simple fix we can support in the next release. Once available, your `~/.ideavimrc` would look something like this:

```
map <C-f> :action AceLineActionRight<CR>
map <C-f> :action AceLineActionLeft<CR>
map <C-f> :action AceLineActionFirst<CR>
```

In the meantime, you can always use `AceLineAction` (<kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>;</kbd> by default), which tags the first, last and first non-whitespace characters of every line at once. Not sure if that's what you want?

**Comment by excetara2** [comment]
Yeah I know about that shortcut but often I just hit `ctrl + ;` then think I want to go to a line number, so I was just wondering if that was possible. I will just wait for the next release and add the config file.

**Comment by breandan** [comment]
This was implemented in by @chylex in `3.7.0`. All of the AceJump shortcuts, including the line modes should now be bound to IDE actions and can be configured under `Settings | Keymap | Plugins | AceJump`.
